contrib/golang/filters/http/test/golang_filter_fuzz.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package envoy.extensions.filters.http.golang;
+
+import "contrib/envoy/extensions/filters/http/golang/v3alpha/golang.proto";
+import "test/fuzz/common.proto";
+import "envoy/config/core/v3/base.proto";
+import "validate/validate.proto";
+
+// We only fuzz a single request per iteration.
+message GolangFilterTestCase {
+  envoy.extensions.filters.http.golang.v3alpha.Config config = 1
```

Comment:
One thing to consider is what is the threat model that you'd like this fuzzer to address.
Filter config is typically considered trusted (coming from a trusted domain). Our previous experience shows that fuzzing the config raises false-errors - errors that should not actually occur.
This is ok, but reduces the ability of the fuzzer to detect data-plane issues.
Consider using pre-defined configs with the same `request_data`.

Reply:
Sorry for the late, busy with other stuff.

I already removed the `config` in this proto, and changed to use a const config for fuzzing.
I'm not sure I understand the `pre-defined` configs, please correct me if I'm wrong, thanks.